webpack.config.js

```diff
@@ -120,7 +120,7 @@ function getWebpackConfig( { externalizeWordPressPackages = false } = {}, argv )
 		entry: { build: [ path.join( __dirname, 'client', 'boot', 'app' ) ] },
 		profile: shouldEmitStats,
 		mode: isDevelopment ? 'development' : 'production',
-		devtool: isDevelopment ? '#eval' : process.env.SOURCEMAP || false, // in production builds you can specify a source-map via env var
+		devtool: process.env.SOURCEMAP || ( isDevelopment ? '#eval' : false ), // in production builds you can specify a source-map via env var
```

Comment:
Maybe remove the "in production builds" from this comment as it no longer applies?

Reply:
Indeed, good catch! Removed in c87cec6a57694c04b5627f4631fb729f574baf64